applications/StructuralMechanicsApplication/custom_strategies/custom_schemes/explicit_central_differences_scheme.hpp

```diff
@@ -437,6 +453,11 @@ class ExplicitCentralDifferencesScheme
         // The array of nodes
         NodesArrayType& r_nodes = rModelPart.Nodes();
 
+        /// Working in 2D/3D
+        const auto it_elem_begin = rModelPart.ElementsBegin();
+        const auto& r_geom = it_elem_begin->GetGeometry();
+        const SizeType dim(r_geom.WorkingSpaceDimension());
```

Comment:
why do you retrieve it again here? Isn't this the same as the function-argument?

Reply:
still open